pandas/_libs/src/vendored/ujson/python/JSONtoObj.c

```diff
@@ -38,9 +38,11 @@ Numeric decoder derived from TCL library
 
 // Licence at LICENSES/ULTRAJSON_LICENSE
 
-#include "pandas/vendored/ujson/lib/ultrajson.h"
 #define PY_SSIZE_T_CLEAN
+// clang-format off
```

Comment:
I think you can remove these clang-format comments now too (I assume they were flagging the multiple includes of the header)

Reply:
Unfortunately not. `clang-format` changes the order of imports here, so without the comment the change would be reverted by pre-commit.

Reply:
Ah OK...that makes sense. Thanks.

I see CI is broken now but I believe that's unrelated